cwms-servers/CWMS_HOME/bin/stopLoginServerLogWatcher

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# stopAccessLogMonitor.sh
+
+# Read the saved PID
+pid=$(cat "$CWMS_HOME/dated/logs/loginServer/accessLogMonitor.pid")
+
+# Kill the process
+kill $pid
```

Comment:
this is safe and wont leave things in an indeterminate state?

Reply:
The process never writes and the read access to the log files will get cleaned up by the kill command.

Reply:
Added a follow-up issue to handle the robust exit: https://github.com/USACE-WaterManagement/cwms-logging/issues/3

Basic research suggests adding a SIGTERM handler will allow the kill command to run a cleanup method within python: https://docs.python.org/3/library/signal.html